entity-framework/ef6/modeling/code-first/data-annotations.md

```diff
@@ -41,15 +41,15 @@ I’ll demonstrate code first DataAnnotations with a simple pair of classes: Blo
     }
 ```
 
-As they are, the Blog and Post classes conveniently follow code first convention and required no tweaks to help EF work with them. But you can also use the annotations to provide more information to EF about the classes and the database that they map to.
+As they are, the Blog and Post classes conveniently follow code first convention and require no tweaks to enable EF compatability. However, you can also use the annotations to provide more information to EF, about the classes and the database to which they map.
```

Comment:
I would omit the comma after EF.
Also I prefer "that they map to" as it sounds more like typical spoken English than "to which they map"

Reply:
I also agree on the comma. I prefer to "to which they map" as it is more proper, but I understand your concern here. I can revert that change.

Reply:
Perhaps Diego will have a decision on the phrasing.